cmd/plume/prerelease.go

```diff
@@ -91,6 +94,19 @@ func init() {
 }
 
 func runPreRelease(cmd *cobra.Command, args []string) error {
+	getClient := func() (client *http.Client, err error) {
+		if gceJSONKeyFile != "" {
+			b, err := ioutil.ReadFile(gceJSONKeyFile)
+			if err != nil {
+				plog.Fatal(err)
+			}
+			client, err = auth.GoogleClientFromJSONKey(b)
```

Comment:
I wasn't sure how to cleanly unify this in `auth` between the various command options, but I've added a command that uses a global option in `plume` so all subcommands support JSON keys.